tests/services/detectionlists/test_departing_employee.py

```diff
@@ -266,85 +218,103 @@ def test_get_all_posts_expected_data_with_non_default_values(
             and posted_data["srtDirection"] == "DESC"
         )
 
-    def test_set_alerts_enabled_posts_expected_data(
+    def test_get_posts_expected_data_to_expected_url(
```

Comment:
I combined a couple tests into one, to fit the pattern we typically use `posts_expected_data_to_expected_url()` rather than two tests, one that verifies the data and one that verifies the URL